detekt-rules/src/main/kotlin/OrtCopyrightHeader.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.detekt
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+
+import org.jetbrains.kotlin.psi.KtFile
+
+private const val NUMBER_OF_HEADER_LINES = 2
+private const val COPYRIGHT_STATEMENT = "Copyright (C) "
+
+class OrtCopyrightHeader : Rule() {
+    override val issue = Issue(
+        javaClass.simpleName,
+        Severity.Style,
+        "Reports files that do not contain ORT's Copyright header",
+        Debt.FIVE_MINS
+    )
+
+    override fun visitKtFile(file: KtFile) {
+        super.visitKtFile(file)
+
+        val copyrightLine = file.text.lineSequence().take(2).find { line ->
+            line.contains(COPYRIGHT_STATEMENT)
+        }
```

Comment:
Why not just check the second line, as the first should never contain a copyright statement?

Reply:
For the future, should we extend the check to also verify that the license is Apache-2.0?

Reply:
> Why not just check the second line, as the first should never contain a copyright statement?

To be frank, just because it was easier to implement it elegantly like this.

> For the future, should we extend the check to also verify that the license is Apache-2.0?

Maybe. Basically, this check is supposed to ensure that a header is added to each file, and we simply use the Copyright line as a marker to check for the presence of the header as a whole. Whether the Copyright year, holder, and license is actually also present and correct can and should be done as part of the manual review, IMO.

Reply:
> > Why not just check the second line, as the first should never contain a copyright statement?
> 
> To be frank, just because it was easier to implement it elegantly like this.

I've changed this now to only look at exactly the expected line.